CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -16,6 +16,11 @@ public class HarvestGatewayTests
         private const string HarvestAccountId = "123456";
         private const string UserAgent = "The Wolves";
 
+        private static readonly string Roles =
```

Comment:
Perhaps renaming this to something more suitable such as `DeveloperRoles`.

Reply:
```suggestion
        private static readonly string DeveloperRoles =
```

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/31/commits/62e3a16f318f93fe515d946723f0bbe769433150.